helpdesk_mgmt_timesheet/views/helpdesk_ticket_view.xml

```diff
@@ -261,4 +261,14 @@
             </field>
         </field>
     </record>
+    <record id="view_task_form2_inherited" model="ir.ui.view">
```

Comment:
This view should be placed in its own `project.task` xml file.

Also, not tested and not sure, this could lead to security issues, when user is not allowed to Helpdesk app. Could you check?

Reply:
Checking!

Reply:
Made the following changes: 
- moved the view to another xml file related with project.task. 
- added a group restriction to helpdesk user so that only users in that group can view the field.
@dalonsod